components/button/button.config.js

```diff
@@ -1,6 +1,42 @@
 module.exports = {
   "title": "Button",
   "default": "primary",
-  "status": "wip",
   "collated": true,
+  "status": "ready",
+  "variants": [{
+    "name": "primary",
+    "label": "Primary",
+    "context": {
+      "btn-classes": "btn--primary",
+      "label": "Submit"
+    },
+  }, {
+    "name": "primary-lg",
+    "label": "Primary (Large)",
+    "context": {
+      "btn-classes": "btn--primary btn--lg",
+      "label": "Start survey"
+    },
+  }, {
+    "name": "secondary",
+    "label": "Secondary - bordered",
+    "context": {
+      "btn-classes": "btn--secondary btn--border",
+      "label": "Add a person"
+    },
+  }, {
+    "name": "link",
+    "label": "Link",
+    "context": {
+      "btn-classes": "btn--link",
```

Comment:
I thought Buttons styles as links was something we wanted to avoid?

Reply:
Yeh but it is a current pattern. This PR is about implementing buttons in a better way to the pattern library